topic 3/src/main/java/org/globant/topic3/ex02/CrudServiceImp.java

```diff
@@ -0,0 +1,114 @@
+package org.globant.topic3.ex02;
+
+import java.util.Scanner;
+
+public class CrudServiceImp implements CrudService {
+	protected CrudServiceImp() {
+	};
+
+	public void create() {
+		UserObject[] user = CrudTest.getUser();
+
+		// Create user
+
+	
+		String newUser = null, newPass = null, newName = null;
+
+		System.out.println("insert new user: ");
+		Scanner scanUser = new Scanner(System.in);
+		newUser = scanUser.next();
+
+		System.out.println("insert new pass: ");
+		Scanner scanPass = new Scanner(System.in);
+		newPass = scanPass.next();
+
+		System.out.println("insert new name: ");
+		Scanner scanName = new Scanner(System.in);
+		newName = scanName.next();
+
+		for (int i = 0; i <= 4; i++) {
+			if (user[i].id == null) {
+				user[i].id = newUser;
+				user[i].pass = newPass;
+				user[i].name = newName;
+				System.out.println("the user was registered successfuly");
+				break;
+			}
+		}
+
+		for (int i = 0; i <= 4; i++) {
+			System.out.println(user[i].id + user[i].pass);
+		}
+	}
+
+	public void read() {
+		UserObject[] user = CrudTest.getUser();
+		String userRead=null;
+		
+		System.out.println("insert the name of the user you wnat to see: ");
```

Comment:
wnat -> want, You ask enter a name, and then make the equal by id, please fix this